library/network/src/modules/CWMFirewallInterfaces.rb

```diff
@@ -1095,7 +1095,7 @@ def CreateOpenFirewallWidget(settings)
       end
       firewall_settings = VBox(
         Frame(
-          _("Firewall Settings"),
+          _("Firewall Settings for #{SuSEFirewall.firewall_service}"),
```

Comment:
This will not work, see, e.g. this below, as a working example:

    _("Service with name '%{service_name}' does not exist") % { service_name: service_name }